big_o/big_o.py

```diff
@@ -95,7 +100,8 @@ def infer_big_o_class(ns, time, classes=ALL_CLASSES, verbose=False):
     for class_ in classes:
         inst = class_()
         residuals = inst.fit(ns, time)
-        fitted[inst] = residuals
+        fitted[inst] = residuals if not return_raw_data else {
+            'residuals': residuals, 'measures': ns, 'times': time}
```

Comment:
There's no need to change this function, since `ns` and `time` are passed in and not modified.

Please modify `big_o` instead, so that it adds the extra information if the flag is `True`. In this way, all the changes are localized in one place.

Reply:
I applied your suggestions. To your point, since measures and times are not changed I thought it would be better to add them once to the fitted dict rather than to duplicate them on each class entry.